bot/util/logger.py

```diff
@@ -0,0 +1,47 @@
+# importing modules
+from os import getcwd
+from time import time
+import logging
+
+# config the logging module
+path = getcwd() + "/logs"
+logging.basicConfig(filename=path + "current.log", level=logging.INFO, format='%(asctime)s - [%(levelname)s]: %(message)s')
+
+def init():
+    with open(path + "current.log", "r") as f:
+        data = f.read()
+        with open(path + "current.log", "w") as f1:
+            if data != "":
+                ts = data.split("\n")[0]
+                print(ts)
+                with open(path + f"{ts}.log", "w") as file:
+                    file.write(data)
+            f1.write(str(int(time())) + "\n")
+
+
+# implementing the logger class
+class Logger:
+    def __init__(self, s):
+        self.s = s
+        self.start()
+    
```

Comment:
Perhaps this could be made as a simple variable so it's more clear in the rest of the code.
```suggestion
    self.cog_name = s.__class__.__name__
```